<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartWindow</class>
 <widget class="QMainWindow" name="ChartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>1231</width>
      <height>821</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
	
	border-image: url(:/IMG_2027.JPG);}</string>
    </property>
    <widget class="QPushButton" name="commitPushButton">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>110</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{	
	background-color: rgb(141, 141, 141,120);
	color: rgb(255, 85, 0);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 12pt &quot;黑体&quot;;
	border-radius:3px;
}
QPushButton:pressed{
padding-top:3px;
padding-left:4px;
}</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 12pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;Copperplate Gothic Bold&quot;;</string>
     </property>
     <property name="text">
      <string>~</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(141, 141, 141,120);
	color: rgb(255, 255, 255);

border:1px solid #e1e0e0;
	border-color: rgb(0, 170, 255);

	font: 12pt &quot;黑体&quot;;
	border-radius:3px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>72</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 12pt &quot;04b_21&quot;;</string>
     </property>
     <property name="text">
      <string> 时间</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>180</y>
       <width>1051</width>
       <height>571</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/IMG_2027.JPG);

height:50px; margin-right: 2px; margin-bottom:-2px;

	border-radius:3px;
	color:rgb(0,0,0);

</string>
     </property>
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="temperatureTab">
      <property name="styleSheet">
       <string notr="true">
</string>
      </property>
      <attribute name="title">
       <string>温度</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QtCharts::QChartView" name="temperatureWidget" native="true">
         <property name="styleSheet">
          <string notr="true">ackground-color:rgba(244,244,244,2);
border-image: url(:/IMG_2027.JPG);
</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/IMG_2027.JPG);</string>
      </property>
      <attribute name="title">
       <string>心率</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QtCharts::QChartView" name="heartRateWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>血压</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QtCharts::QChartView" name="bloodPretureWidget" native="true">
         <property name="styleSheet">
          <string notr="true">border-image: url(:/IMG_2027.JPG);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>血氧</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QtCharts::QChartView" name="bloodOxygenWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>血糖</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QtCharts::QChartView" name="bloodGluseWidget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1227</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
